templates/firststeps.tpl.php

```diff
@@ -1,21 +1,34 @@
 <?php
-/*
- * @file Template file for Quickstart/First Steps block.
+/**
+ * @file
+ * Template file for Quickstart/First Steps block.
  */
 
-/*
- * Variables
+/**
+ * Variables.
  */
-$user_login_text = l(t('Log in with your SUNetID.'), 'sites/default/webauth/login');
-$webauth_allow_local = variable_get('webauth_allow_local');
-$password_reset = l(t('password reset form'), 'user/password');
+
+// Authentication scheme defaults to SAML.
+$sso_authentication_scheme = 'saml';
+if (module_exists('webauth') && !module_exists('stanford_simplesamlphp_auth')) {
+
+  $sso_authentication_scheme = 'webauth';
+}
+elseif (!module_exists('webauth') && !module_exists('stanford_simplesamlphp_auth')) {
+  $sso_authentication_scheme = 'local';
+}
+$user_login_text = l(t('Log in with your SUNetID'), 'sso/login') . '.';
+if ($sso_authentication_scheme == "webauth") {
+  $user_login_text = l(t('Log in with your SUNetID'), 'sites/default/webauth/login') . '.';
```

Comment:
You are allowed punctuation in the t() function. No need to concatenate the period after.

Reply:
I don't want the period in the hyperlink.